velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -75,13 +75,40 @@ static std::vector<std::string> kRangeFrameClauses = {
 
 /// Common set of window function frame clauses in ROWS mode, with current row,
 /// unbounded preceding, and unbounded following frame combinations.
-static std::vector<std::string> kRowsFrameClauses = {
+static std::vector<std::string> kUnboundedRowsFrameClauses = {
     "rows unbounded preceding",
     "rows current row",
     "rows between current row and unbounded following",
     "rows between unbounded preceding and unbounded following",
 };
 
+/// Common set of window function frame clauses in ROWS mode with k preceding
+/// and k following frame bounds, where k is a constant integer.
+static std::vector<std::string> kConstantKBoundRowsFrameClauses = {
```

Comment:
We should add frames like "5 preceding and 1 preceding", "1 following and 5 following" to this list.

Reply:
Thanks for the suggestion, since these combinations would result in empty frames, they are added in a separate PR (#2815). Will add these test cases to the empty frames PR. Please let me know if empty frame changes should be combined with this PR.